java/device/iothub-java-client/src/main/java/com/microsoft/azure/iothub/transport/mqtt/MqttIotHubConnection.java

```diff
@@ -346,5 +348,24 @@ private void updateConnectionOptions(String userName, String userPassword)
         this.connectionOptions.setMqttVersion(mqttVersion);
         this.connectionOptions.setUserName(userName);
         this.connectionOptions.setPassword(userPassword.toCharArray());
+        this.setEnabledProtocal(enabledProtocols);
+    }
```

Comment:
Would it be possible to add "setEnabledProtocol" as a member of connectionOptions as others are done above ?
[Nit]  Also please correct the spelling from setEnabledProtocal to setEnabledProtocol